src/lib/utils/range.ts

```diff
@@ -0,0 +1,26 @@
+type RangeOptions = {
+  start?: number;
+  end: number;
+  step?: number;
+};
+
+function range({ start = 0, end, step = 1 }: RangeOptions): number[] {
+  // We can't divide by 0 so we re-assign the step to 1 if it happens.
+  const limitStep = step === 0 ? 1 : step;
+
+  // In some cases the array to create has a decimal length.
+  // We therefore need to round the value.
+  // Example:
+  //   { start: 1, end: 5000, step: 500 }
+  //   => Array length = (5000 - 1) / 500 = 9.998
+  const arrayLength = Math.round((end - start) / limitStep);
+
+  return Array.apply(null, Array(arrayLength)).map((_, current) =>
+    // If the step is 0, the values shouldn't get incremented.
+    // Example:
+    //   range({ start: 1, end: 4, step: 0 }) => [1, 1, 1]
+    step === 0 ? start : (start + current) * step
```

Comment:
do we really need step 0 in our code? I'd rather see that replaced by `Array(length).fill(x)`

Reply:
I've looked around, and only see it if someone passes `step:  0` which doesn't really make sense?

Reply:
That's to be completely in sync with the previous Lodash behavior. I bet nobody used `{ step: 0 }` in our widgets... I'm OK to remove this.

Note that `Array.prototype.fill` is not supported by IE.

Reply:
let's remove this, maybe add an error on `step: 0` on the range constructor?